sql/updates/post-update.sql

```diff
@@ -132,3 +132,20 @@ FROM _timescaledb_catalog.hypertable h
 INNER JOIN _timescaledb_catalog.dimension d ON (d.hypertable_id = h.id)
 WHERE d.interval_length IS NULL; 
 DROP FUNCTION _timescaledb_internal.update_dimension_partition;
+
+-- Report warning when partial aggregates are used
+DO $$
+DECLARE
+  cagg_name text;
+BEGIN
+    FOR cagg_name IN
+      SELECT
+        attrelid::regclass::text
+      FROM _timescaledb_catalog.continuous_agg cagg
+      INNER JOIN pg_attribute att ON (
+        att.attrelid = format('%I.%I',cagg.user_view_schema,cagg.user_view_name)::regclass)
+      WHERE cagg.finalized = false
+    LOOP
+      RAISE WARNING 'Continuous Aggregate: % with old format will not be supported with PG15. You should upgrade to the new format', cagg_name;
```

Comment:
Shouldn't this WARNING be emitted only if we're upgrading TimescaleDB using PG15??

Reply:
Plan is to deprecate with new Timescaledb extension, irrespective of what PG version customer is using. This gives us more time to let users, to migrate to new formats. And we are not sure when will customer move to PG15.

And with PG15 we don't want customers to create cagss with old formats.